pkg/translator/opencensus/internaldata/metrics_to_oc.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package internaldata
+package opencensus
```

Comment:
should these files all be up one level?

Reply:
Outside of the internaldata folder ?

Reply:
yes, in `pkg/translator/opencensus`.  Would align the package name with the import path and avoid an extra element in the import path that adds no value.

Reply:
I have just pushed a commit that addresses this